dhcp/config.ml

```diff
@@ -0,0 +1,21 @@
+open Mirage
+
+let main = foreign "Unikernel.Main" (console @-> kv_ro @-> network @-> clock @-> job)
+
+let disk = crunch "files"
+
+(* TODO: workaround a bug in the command-line tool by adding Clock
+   for Unix (this is pulled in as an implicit dependency in Xen) *)
```

Comment:
This is ... ancient. I think @haesbaert copied the config.ml from stack, where this was present. It's not valid anymore.